src/Contracts/MockClient.php

```diff
@@ -104,7 +104,7 @@ public function assertNotSent(string|callable $request): void;
      * Assert JSON data was sent
      *
      * @param string $request
-     * @param array $data
+     * @param array<mixed, mixed> $data
```

Comment:
Keys can only really be `int|string` *1. To type hint array keys, you can use `array-key` for those.

1. unless you use `\WeakMaps`, at which point objects are turned into internal hashes.

```suggestion
     * @param array<array-key, mixed> $data
```